recipes/lua-5.1/build.sh

```diff
@@ -0,0 +1,11 @@
+if [ `uname` == Darwin ]; then
+	make macosx INSTALL_TOP=$PREFIX
+elif [ `uname` == Linux ]; then
+	make linux INSTALL_TOP=$PREFIX MYCFLAGS="-I$PREFIX/include -L$PREFIX/lib -DLUA_USE_LINUX -DLUA_USER_DEFAULT_PATH='\"$PREFIX/\"'" MYLDFLAGS="-L$PREFIX/lib -Wl,-rpath=$PREFIX/lib"
+fi
+make install INSTALL_TOP=$PREFIX
+if [ `uname` == Darwin ]; then
+	make macosx test
+elif [ `uname` == Linux ]; then
+	make linux test
+fi
```

Comment:
Can you run the tests before the `make install`? If so you can put those lines inside the if-clause above, no?